### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by acuna-public** [willbprog127/spiritvnc-fltk#5]
Hello! Thanks for this. But is it support Windows, because fltk is cross-platform?

**Comment by willbprog127** [comment]
Greetings!  I have thought about a Windows version.  I will do some preliminary work on it and see if it's feasible.

Thanks! 👍

**Comment by acuna-public** [comment]
Hello, sorry for my long answer. That's sounds nice) But what the problem with Windows? I plan to use fltk too in my project, but I thought that the main idea of it is cross-platform, or I've missed something?)

**Comment by willbprog127** [comment]
> Hello, sorry for my long answer. That's sounds nice) But what the problem with Windows? I plan to use fltk too in my project, but I thought that the main idea of it is cross-platform, or I've missed something?)

Hi again 😄 

Last time I tried, FLTK was a big pain to build on Windows compared to other platforms (Linux, BSD, macOS).  I don't think it's impossible, it will just take more work than on *nix-type systems.  Also it depends if libssh2 and libvncclient will build OK on Windows or if they will need tweaking.

Thanks! 👍

**Comment by acuna-public** [comment]
Wow, I'm so confused) But do you know a lightweight cross-platform library, because I want to support several systems? I hoped on fltk because its final projects is extremely lightweight...

**Comment by willbprog127** [comment]
> Wow, I'm so confused) But do you know a lightweight cross-platform library, because I want to support several systems? I hoped on fltk because the final projects on it are extremely lightweight...

Yes, FLTK is very lightweight and this is one reason why I like it.

Just because *I* had a problem building FLTK on Windows does not mean you will.  Don't abandon your plans just because of what I say. 😉

**Comment by acuna-public** [comment]
Aaaaahh, youuuu, I get it, thanks)))
